source/standards/managing-sensitive-information.html.md.erb

```diff
@@ -0,0 +1,397 @@
+[2012-coding-in-open]: https://gds.blog.gov.uk/2012/10/12/coding-in-the-open/
+[2017-why-we-open]: https://mojdigital.blog.gov.uk/2017/02/21/why-we-code-in-the-open/
+[2017-benefits-of-open]: https://gds.blog.gov.uk/2017/09/04/the-benefits-of-coding-in-the-open/
+[2017-dont-be-afraid]: https://technology.blog.gov.uk/2017/09/27/dont-be-afraid-to-code-in-the-open-heres-how-to-do-it-securely/
+[2018-barriers-to-open]: https://technology.blog.gov.uk/2018/11/05/11-barriers-to-coding-in-the-open-and-how-to-overcome-them/
+[moj-open-blog]: https://technology.blog.gov.uk/2018/01/05/how-making-our-deployment-code-open-improved-our-workflow/
+[nhs-service-standards]: https://service-manual.nhs.uk/standards-and-technology/service-standard-points/12-make-new-source-code-open
+[government-security-classifications-policy]: https://www.gov.uk/government/publications/government-security-classifications/government-security-classifications-policy-html
+[definition-of-official]: https://www.gov.uk/government/publications/government-security-classifications/government-security-classifications-policy-html#official
+[definition-of-secret]: https://www.gov.uk/government/publications/government-security-classifications/government-security-classifications-policy-html#secret
+[definition-of-top-secret]: https://www.gov.uk/government/publications/government-security-classifications/government-security-classifications-policy-html#top-secret
+[application-of-sensitive]: https://www.gov.uk/government/publications/government-security-classifications/government-security-classifications-policy-html#applying-the--sensitive-marking
+[handling-instructions]: https://www.gov.uk/government/publications/government-security-classifications/government-security-classifications-policy-html#handling-instructions
+[github-repos]: https://github.com/orgs/govuk-one-login/repositories
+[github-repos-private]: https://github.com/orgs/govuk-one-login/repositories?q=visibility%3Aprivate
+[github-repos-internal]: https://github.com/orgs/govuk-one-login/repositories?q=visibility%3Ainternal
+[github-repos-public]: https://github.com/orgs/govuk-one-login/repositories?q=visibility%3Apublic
+[github-visibility]: https://docs.github.com/en/enterprise-cloud@latest/repositories/creating-and-managing-repositories/about-repositories#about-repository-visibility
+[github-visibility-resource]: https://resources.github.com/learn/pathways/administration-governance/essentials/manage-your-repository-visibility-rules-and-settings/
+[when-code-should-be-open-or-closed]: https://www.gov.uk/government/publications/open-source-guidance/when-code-should-be-open-or-closed
+[service-manual-open]: https://www.gov.uk/service-manual/technology/making-source-code-open-and-reusable
+[code-owners]: https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
+[config-repo]: https://github.com/govuk-one-login/configuration
```

Comment:
```suggestion
[config-repo]: https://github.com/govuk-one-login/configuration

```

New lines are good, for reasons I can never remember but have come to dogmatically believe